go/go.mod

```diff
@@ -12,20 +12,38 @@
 
 module arrow-flight-client-example
 
-go 1.16
+go 1.22.0
+
+toolchain go1.23.3
 
 require (
-	github.com/apache/arrow/go/arrow v0.0.0-20211112161151-bc219186db40
+	github.com/apache/arrow-go/v18 v18.0.0
 	github.com/docopt/docopt-go v0.0.0-20180111231733-ee0de3bc6815
-	github.com/google/flatbuffers v2.0.6+incompatible // indirect
-	github.com/klauspost/compress v1.15.0 // indirect
-	github.com/pierrec/lz4/v4 v4.1.14 // indirect
-	github.com/stretchr/testify v1.7.0
+	github.com/stretchr/testify v1.9.0
 	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616
-	golang.org/x/net v0.0.0-20220225172249-27dd8689420f // indirect
-	golang.org/x/sys v0.0.0-20220310020820-b874c991c1a5 // indirect
-	golang.org/x/tools v0.1.9 // indirect
-	google.golang.org/genproto v0.0.0-20220310185008-1973136f34c6 // indirect
-	google.golang.org/grpc v1.45.0
-	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+	google.golang.org/grpc v1.67.1
+)
+
+require (
```

Comment:
check it, if it doesn't use, remove and reindex

remove go.mod, and next run go mod tidy

Reply:
@valterfrancisco-dremio not deleting go.sum

delete go.sum, and next run go mod tidy, go.sum should appear